### PR TITLE
Make SFN->Lambda trace linking doc collapsable

### PR DESCRIPTION
**Pull request by lym953**
### What does this PR do? What is the motivation?

On this page: https://docs.datadoghq.com/serverless/step_functions/installation/?tab=custom, make this part collapsable:
image

This part is for Step Function -> Lambda trace linking. In a later PR, I'll add a section for Step Function -> Step Function trace linking. I want to make both sections collapsable to avoid making the page too long and scaring our developers.

This PR addresses comments on @cswatt  PR https://github.com/DataDog/documentation/pull/25517. Because she won't be back until Oct 21, I'm creating this new PR.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/serverless/step_functions/installation.md [hunk 1]
```diff
@@ -185,13 +188,19 @@ Alternatively, if you have business logic defined in the payload, you could also
...
 If you have not yet instrumented your Lambda functions to send traces, you can [follow the steps to add the Lambda layer for your preferred runtime][3].
 
+[1]: https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-json-manipulate
+[2]: https://docs.aws.amazon.com/step-functions/latest/dg/input-output-contextobject.html
+[3]: /logs/guide/forwarder/?tab=cloudformation#installation
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Link Step Function traces to nested Step Function traces" level="h4" %}}
+TODO
+{{% /collapse-content %}}
```
Comment (hestonhoffman) [inline content/en/serverless/step_functions/installation.md:198]:
```suggestion
<!--
{{% collapse-content title="Link Step Function traces to nested Step Function traces" level="h4" %}}
TODO
{{% /collapse-content %}} 
-->
```
Commenting this out until there's useful content to present to the user.

File: content/en/serverless/step_functions/installation.md [hunk 2]
```diff
@@ -139,16 +139,19 @@ For developers using [Serverless Framework][4] to deploy serverless applications
 <div class="alert alert-warning"> If you are using a different instrumentation method such as Serverless Framework or datadog-ci, enabling autosubscription may create duplicated logs. Choose one configuration method to avoid this behavior.</div>
 
 4. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>`, `service:<SERVICE_NAME>`, and `version:<VERSION>` tags. The `env` tag is required to see traces in Datadog, and it defaults to `dev`. The `service` tag defaults to the state machine's name. The `version` tag defaults to `1.0`.
-5. For Node.js and Python runtimes, you can link your Step Function traces to Lambda traces. On the Lambda Task, set the `Parameters` key with the following: 
+5. Link your Step Function traces to downstream Lambda traces or nested Step Function traces:
```
Comment (hestonhoffman) [inline content/en/serverless/step_functions/installation.md:142]:
```suggestion
5. Link your Step Function traces to downstream Lambda traces<!-- or nested Step Function traces-->:
```

**Review by hestonhoffman — CHANGES_REQUESTED** [review]
I've commented out the TODO section below (along with references to it), as we can't publish incomplete docs to our users.

File: content/en/serverless/step_functions/installation.md (hunk 2 above)
Comment (lym953) [reply content/en/serverless/step_functions/installation.md]: Make sense! Fixed.

File: content/en/serverless/step_functions/installation.md (hunk 1 above)
Comment (lym953) [reply content/en/serverless/step_functions/installation.md:199]: Make sense! Fixed.

**Review by hestonhoffman — APPROVED** [review]

**Comment by hestonhoffman** [comment]
@lym953 Looks good. Are we good to publish?

**Pull request merged**